ptolemy-py/tests/test_trace.py

```diff
@@ -0,0 +1,39 @@
+"""Test Trace functionality."""
+
+from uuid import uuid4
+import pytest
+from ptolemy_client.client.mock import MockPtolemy
+from ptolemy_client.tier import Tier
+
+@pytest.fixture
+def client() -> MockPtolemy:
+    return MockPtolemy()
+
+def test_trace_subject_id(client: MockPtolemy):
+    trace = client.trace(
+        uuid4(),  # subject_id
+        "my_system_trace",
+        parameters={"foo": "bar"},
+        version="0.0.1",
+        environment="DEV",
+    )
+
+    assert trace.subject_id == trace.parent_id
+    assert trace.tier == Tier.SYSTEM
+
+def test_trace_child(client: MockPtolemy):
```

Comment:
I don't think you need MockPtolemy, you can use `unittest.mock` to replace the methods you don't want triggering.